### PR TITLE
WishList Feature 

### PR DESCRIPTION
**Pull request by manisaiprasad**

### CODE REVIEW

File: client/src/components/Cart/ProductTable.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+  );
+}
```
Comment (iMagesh) [inline client/src/components/Cart/ProductTable.js:95]: This file seems to have a lot of JSX, can we split this into multiple components?

File: client/src/components/WishList/WishList.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+  useEffect(() => {
+    getWishlistItems();
+  }, []);
+  // delete item from cart
+  const deleteItem = (id) => {
+    if (window.confirm("Are you sure you want to delete this item?")) {
+      axios
+        .delete(`/wishlist/${id}`, {
+          headers: {
```
Comment (iMagesh) [inline client/src/components/WishList/WishList.js:37]: this headers object is repeated in many files in multiple places right? can we move this to a separate helper file and import it everywhere as a constant?

**Comment by iMagesh** [comment]
@manisaiprasad are you done with changes? can i merge?

**Comment by iMagesh** [comment]
@manisaiprasad you have some conflicts now, we can merge once you resolve it

**Comment by manisaiprasad** [comment]
Resolved Conflicts @iMagesh

**Pull request merged**